core/include/traccc/seeding/detail/lin_circle.hpp

```diff
@@ -48,37 +48,10 @@ struct lin_circle {
     const scalar& V() const { return m_V; }
 };
 
-/// Container of lin_circle belonging to one detector module
-template <template <typename> class vector_t>
-using lin_circle_collection = vector_t<lin_circle>;
+/// Declare all lin_circle collection types
+using lin_circle_collection_types = collection_types<lin_circle>;
 
-/// Convenience declaration for the lin_circle collection type to use in host
-/// code
-using host_lin_circle_collection = lin_circle_collection<vecmem::vector>;
-
-/// Convenience declaration for the lin_circle collection type to use in device
-/// code
-using device_lin_circle_collection =
-    lin_circle_collection<vecmem::device_vector>;
-
-/// Convenience declaration for the lin_circle container type to use in host
-/// code
-using host_lin_circle_container = host_container<unsigned int, lin_circle>;
-
-/// Convenience declaration for the lin_circle container type to use in device
-/// code
-using device_lin_circle_container = device_container<unsigned int, lin_circle>;
-
-/// Convenience declaration for the lin_circle container data type to use in
-/// host code
-using lin_circle_container_data = container_data<unsigned int, lin_circle>;
-
-/// Convenience declaration for the lin_circle container buffer type to use in
-/// host code
-using lin_circle_container_buffer = container_buffer<unsigned int, lin_circle>;
-
-/// Convenience declaration for the lin_circle container view type to use in
-/// host code
-using lin_circle_container_view = container_view<unsigned int, lin_circle>;
+/// Declare all lin_circle container types
+using lin_circle_container_types = container_types<unsigned int, lin_circle>;
```

Comment:
Let's not even declare this type. It doesn't seem to be used anywhere. (And the first argument of the template seems to be a dummy anyway.)